scripts/sensorshow

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+
+"""
+    Script to show Voltage and Current Sensor status.
+"""
+from tabulate import tabulate
+from swsscommon.swsscommon import SonicV2Connector
+from natsort import natsorted
+import argparse
+
+
+header = ['Sensor', 'Voltage(mV)', 'High TH', 'Low TH', 'Crit High TH', 'Crit Low TH', 'Warning', 'Timestamp']
```

Comment:
`'Voltage(mV)'` should be something generic in this header definition, as it is replaced with the explicit type when needed further down. Perhaps `'Value'`?

Reply:
ack